SU-SWS/ood-giving-campaign
# GIVCAMP-88 | data card

# READY FOR REVIEW

# Summary
- Add data card with animated counter option

# Review By (Date)
- Retro

# Criticality
- 4

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to storyblok test/test-initiative-template page
2. Select the PR245 preview URL in the visual editor
3. Scroll down to see data cards in dark and light mode.
![Screenshot 2024-03-12 at 1 45 22 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/f19a75cc-c550-4134-905d-7572524941f5)
![Screenshot 2024-03-12 at 1 45 28 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/76545069-47a3-4cb1-a96e-dda9494601ca)

4. Check that the cards with number counter enabled, the number in the heading counts up once the card comes into view.

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-88
